app/cdap/components/RecipeList/RecipeTableRow.tsx

```diff
@@ -165,13 +165,17 @@ export const RecipeTableRow = ({
 
   return (
     <>
-      <div className="grid-row" onClick={!showAllColumns ? selectRecipeHandler : undefined}>
+      <div
+        className="grid-row"
+        onClick={!showAllColumns ? selectRecipeHandler : undefined}
+        data-testid={`${recipe.recipeName}-recipe-row`}
```

Comment:
please convert the _recipe.recipeName_ to _hyphenated lowercase string_

Reply:
converted recipe.recipeName to hyphenated lowercase string